models/user.go

```diff
@@ -1360,6 +1360,18 @@ func SearchUsers(opts *SearchUserOptions) (users []*User, _ int64, _ error) {
 		Find(&users)
 }
 
+//SearchUsersAPI takes options i.e keyword and part of user name to search
+//it returns all the results found and the number of total results.
+func SearchUsersAPI(opts *SearchUserOptions) (users []*User, _ int64, _ error) {
```

Comment:
Since you don't really use the return values you initialize, please don't use any since you're still initializing `users`